packages/default-plugins/tests/ecdsa.test.ts

```diff
@@ -54,10 +53,52 @@ const testVectors = [
 
 describe("ecdsa did:key", () => {
   it("derives the correct DID from the JWK", async () => {
-    for(const vector of testVectors) {
+    for (const vector of testVectors) {
       const keypair = await EcdsaKeypair.importFromJwk(vector.jwk)
       const did = keypair.did()
       expect(did).toEqual(vector.id)
     }
   })
-})
\ No newline at end of file
+})
+
+describe("import and exporting a key", () => {
+  let exportableKeypair: EcdsaKeypair;
+  let nonExportableKeypair: EcdsaKeypair;
+
+  beforeAll(async () => {
+    exportableKeypair = await EcdsaKeypair.create({ exportable: true })
+    nonExportableKeypair = await EcdsaKeypair.create({ exportable: false })
+  })
+
+  it("can export a key using jwk", async () => {
+    const exported = await exportableKeypair.export()
+    expect(exported.length).toBeGreaterThan(0)
+  })
+
+  it("won't export a non exportable keypar", async () => {
+    await expect(nonExportableKeypair.export())
+      .rejects
+      .toThrow('Key is not exportable')
+  })
+
+  it('Can export a key and re-import from it', async () => {
+    const exported = await exportableKeypair.export()
+
+    const jwk = JSON.parse(exported)
+    const newKey = await EcdsaKeypair.import(jwk)
```

Comment:
Export returns a `string`, but import takes a `PrivateKeyJWK`. Perhaps both should be based around `PrivateKeyJWK`? Or `string`?
I'd be happier with a change of return type in `export()` rather than trying to be backwards-compatible. What do you think?

Reply:
Yeah, it's a bit awkward. I would prefer to normalize this on either `PrivateKeyJWK` or the more general `JsonWebKey`. The former is a bit more descriptive, the latter a bit more interoperable. Any strong feelings here?

Reply:
No strong feelings, but leaning towards `PrivateKeyJWK`, because the string representation is just a `JSON.stringify` away for users. That way users can opt-in to the version that's interoperable, but with worse types.